titus-server-runtime/src/main/java/com/netflix/titus/runtime/jobmanager/gateway/GrpcJobServiceGateway.java

```diff
@@ -100,6 +102,14 @@ public Completable updateJobCapacity(JobCapacityUpdate jobCapacityUpdate) {
         }, configuration.getRequestTimeoutMs());
     }
 
+    @Override
+    public Completable updateJobCapacityWithOptionalAttributes(JobCapacityUpdateWithOptionalAttributes jobCapacityUpdateWithOptionalAttributes) {
```

Comment:
Are we preferring Reactor to Rx types? We already use `Mono<Void>` in this class.

Reply:
That seems to be the direction we are taking.